lib/db.py

```diff
@@ -68,6 +68,17 @@ def fetch_one(self, query, params=None, fetch_mode=None):
     def fetch_all(self, query, params=None, fetch_mode=None):
         return self.__query(query, params=params, return_type='all', fetch_mode=fetch_mode)
 
+    def import_csv(self, filename):
+        with self._pool.connection() as conn:
+            conn.autocommit = True
+            cur = conn.cursor()
+            with open(filename, 'r', encoding='utf-8') as sql_file:
+                sql_script = sql_file.read()
+                for statement in sql_script.split(';'):
+                    if statement.strip():
+                        cur.execute(statement)
```

Comment:
- What happens if there is an exception? I would rollback on exception.
- Where do you commit?

Reply:
This sadly cannot be done. I must set the adapter in autocommit mode. Otherwise it cannot execute this block statement.